### ISSUE TITLE
[v10] Inconsistent header between browsers (application name cut off in IE11)

### ISSUE THREAD

**Issue by ncrawlins** [auth0/lock#478]
Using lock 10 rc2

![lock-issue](https://cloud.githubusercontent.com/assets/291462/16689331/74d2b8c4-44f0-11e6-8af2-cc22068c108c.png)

**Comment by gnandretta** [comment]
@beneliflo can you take a look at this?

**Comment by fiddur** [comment]
For a customer we made this workaround temporarily:

```
.auth0-lock.auth0-lock .auth0-lock-header {
  height: auto !important;
}
```

IE11 seems to include the padding in the height, so the fixed height to 90px eats up the insides.  `auto` seems to be good at least in IE11 and Chrome.

**Comment by gnandretta** [comment]
Fixed on #507
